samples/system-test/spanner.test.js

```diff
@@ -41,624 +41,708 @@ const spanner = new Spanner({
   projectId: PROJECT_ID,
 });
 
-test.before(tools.checkCredentials);
-
-test.before(async () => {
-  const instance = spanner.instance(INSTANCE_ID);
-  const database = instance.database(DATABASE_ID);
-
-  try {
-    await instance.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  try {
-    await database.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  const [, operation] = await instance.create({
-    config: 'regional-us-central1',
-    nodes: 1,
-    labels: {
-      'gcloud-sample-tests': 'true',
-    },
+describe('Spanner', () => {
+  before(async () => {
+    tools.checkCredentials();
+    const instance = spanner.instance(INSTANCE_ID);
+    const database = instance.database(DATABASE_ID);
+
+    try {
+      await instance.delete();
+    } catch (err) {
+      // Ignore error
+    }
+
+    try {
+      await database.delete();
+    } catch (err) {
+      // Ignore error
+    }
+
+    const [, operation] = await instance.create({
+      config: 'regional-us-central1',
+      nodes: 1,
+      labels: {
+        'gcloud-sample-tests': 'true',
+      },
+    });
+
+    try {
```

Comment:
We don't want to wrap this in a try/catch

Reply:
Removed try/catch